SortApp/SortApp.Presentation/App.xaml.cs

```diff
@@ -1,17 +1,26 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Data;
-using System.Linq;
-using System.Threading.Tasks;
 using System.Windows;
+using SortApp.ViewModel;
 
 namespace SortApp.Presentation
 {
-	/// <summary>
-	/// Interaction logic for App.xaml
-	/// </summary>
-	public partial class App : Application
-	{
-	}
+    /// <summary>
+    /// Interaction logic for App.xaml
+    /// </summary>

```

Comment:
It's always the one who have added this code. Even if it's generated by VS - you are responsible for it